doc/api/corepack.md

```diff
@@ -16,6 +16,6 @@ Documentation for this tool can be found on the [Corepack repository][].
 
 Despite Corepack being distributed with default installs of Node.js, the package
 managers managed by Corepack are not part of the Node.js distribution, and
-Corepack itself will no longer be distributed with future versions of Node.js.
+Corepack itself will no longer be distributed with Node.js 25+.
```

Comment:
Using the notation from https://github.com/nodejs/release including the use of the term "release line" to refer to all releases with same major version number:

```suggestion
Corepack itself will no longer be distributed with release lines Node.js 25 and above.
```

You could also write "releases Node.js `>=25.0.0`" using [semver](https://semver.org/) notation.